TabunAva Reborn.user.js

```diff
@@ -918,6 +925,10 @@ function fixStyles() {
   if (GSettings.fixcommentcorners) {
     styleSheet.innerText += ".comment-content { border-radius: 8px; } ";
   }
+  if (GSettings.oldauthor) {
+    styleSheet.innerText += ".comment-info .comment-author.comment-topic-author span { color: #779; } "; // I don't recall actual color, if you know, pls let me know too
```

Comment:
#4b5468

https://gitea.everypony.ru/everypony/tabun/src/commit/889f6e868a7d255a88ee4a4e5804ed32dcf36891/frontend/styles/comments.styl#L121

Reply:
Wow, right, thanks! Gonna change it now.